README.md

```diff
@@ -1,2 +1,27 @@
-# workloads
-Collection of snapshots of WebAssembly workloads
+# WebAssembly Workloads
+
+This repo contains a collection of snapshots of WebAssembly workloads
+served at [webassembly.github.io/workloads](https://webassembly.github.io/workloads).
+
+The goals of the repo are:
+* to provide a common place to share workloads between developers of
+  WebAssembly applications, engines and toolchain
```

Comment:
Something we mentioned on the WebAssembly CG call today was that we don't want to be measuring mostly non-WebAssembly things such as WebGL. We should mention that here.

Reply:
Agreed, added (but in the next bullet)